test/e2e/case40_catalogsource_status_report_test.go

```diff
@@ -0,0 +1,157 @@
+package e2e
+
+import (
+	"encoding/json"
+
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	policyv1 "open-cluster-management.io/config-policy-controller/api/v1"
+	policyv1beta1 "open-cluster-management.io/config-policy-controller/api/v1beta1"
+	"open-cluster-management.io/config-policy-controller/test/utils"
+)
+
+var _ = Describe("Test status reporting for catalogsource", Ordered, func() {
```

Comment:
Is there a problem if we merge this into the big describe in the case38 file? I think it's kind of nice to be able to quickly run all of the OperatorPolicy tests at once right now.

Reply:
Yea I could do that. I was thinking of moving the `check` function to a common file since it's used by a few files, but moving the entire test into one file works too!